### ISSUE TITLE
Dataset without data augmentation?

### ISSUE THREAD

**Issue by CassiniHuy** [lincshunter/TADBoundaryDectector#2]
Hi! The training set provided in this repo is augmented. Could you provide the dataset without augmentation? Thx.

**Comment by CassiniHuy** [comment]
I find the data provided in this repo is not okay. The samples of training set, validation&test set are overlapped.
